objection/utils/patchers/android.py

```diff
@@ -211,6 +213,8 @@ def __init__(self, skip_cleanup: bool = False, skip_resources: bool = False, man
         self.skip_resources = skip_resources
         self.manifest = manifest
 
+        self.architecture = None
```

Comment:
Don't think this is used for anything.

Reply:
I'm using it below in `inject_load_library` for injecting the library with the correct architecture.